### DISCUSSION TITLE
Need help diagnosing performance issue

### DISCUSSION THREAD

**Discussion by gregwinterstein** [gitextensions/gitextensions#10167] [Q&A]
I have started seeing a performance issue with Git Extensions that is only resolved by rebooting. After some time (more than 12 hours), Git Extensions becomes very sluggish. After rebooting, the performance returns to its normal (snappy) performance. While it is sluggish, the Windows task manager does not show much activity.

I am currently using: 
- Git Extensions 4.0.0.15025
- Build e8588e4c048e94d62054911fc09decb8139931f6
- Git 2.37.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET 6.0.8
- DPI 96dpi (no scaling)

The issue occurred in the previous stable version, 3.5.4. I upgraded to 4.0.0.15025 in an attempt to resolve the issue.
The issue started recently (past month or so), long after I had been using version 3.5.4.

I'm attaching GitExtensions Logs from before the reboot (slow), after the reboot (fast), and the repo stats.
[git_extensions_log_20220826_before_reboot.txt](https://github.com/gitextensions/gitextensions/files/9435105/git_extensions_log_20220826_before_reboot.txt)
[git_extensions_log_20220826_after_reboot.txt](https://github.com/gitextensions/gitextensions/files/9435106/git_extensions_log_20220826_after_reboot.txt)
[git_repo_stats.txt](https://github.com/gitextensions/gitextensions/files/9435107/git_repo_stats.txt)

Any suggestions for figuring this out would be appreciated.

**Comment by gerhardol** [comment]
This is a Git issue, unknown what it is.
Six seconds to get the version when Git is not even interacting with the file system is ridicolus...
You will get same performance from the commandline, so not related to GE
But I do not know where the best place to ask about this problem is.

You can try to downgrade Git to see if there is a regression

Log before reboot:
```
2022-08-26T07:04:50.4681861-07:00   6136    32164   UI  0   git --version   
```
